netutils/thttpd/fdwatch.c

```diff
@@ -296,7 +296,7 @@ int fdwatch(struct fdwatch_s *fw, long timeout_msecs)
             {
               /* Yes... save it in a shorter list */
 
```

Comment:
please fix the "Relative file path does not match actual file" as well

Reply:
Done